stack/package-homepage-topic-type.rst

```diff
@@ -0,0 +1,127 @@
+
+   See :doc:`layout-of-doc-directory` for more information.
+
+.. _package-homepage-template:
+
+Starter template
+================
+
+The `stack_package`_ project template includes the `Jinja-formatted template for the module homepage <https://raw.githubusercontent.com/lsst/templates/master/project_templates/stack_package/%7B%7Bcookiecutter.package_name%7D%7D/doc/%7B%7Bcookiecutter.package_name%7D%7D/index.rst>`_.
+
+For an example package named ``example``, the rendered template looks like this:
+
+.. remote-code-block:: https://raw.githubusercontent.com/lsst/templates/master/project_templates/stack_package/example/doc/example/index.rst
+   :language: rst
+
+The next sections describe the key components of the package homepage.
+
+.. _package-homepage-filename:
+
+File name and location
+======================
+
+This file must be named :file:`index.rst` and must be located in a :ref:`package documentation directory <docdir-package-doc-directory>` within a package.
+For example, if the package’s name is ``example``, the full path for the file is :file:`doc/example/index.rst`.
+
+.. _package-homepage-title:
+
+Title
+=====
+
+The title (top-level header) of the package homepage is the packages’s name, without any special formatting (like code literals).
+
+.. note::
+
+   In the future we might include a brief descriptive phrase after the package’s name.
+   There is no guidance to add this descriptive phrase at this time.
+
+The cross-reference target above the title must be name of the package followed by a ``-package`` suffix (see the example from the :ref:`starter template <package-homepage-template>`).
+
+.. _package-homepage-context:
+
+Context paragraph
+=================
+
+Directly after the title, include one or two paragraphs that describe what the module is for.
+The purpose of this content is not to describe the package in detail, or how to use it, but instead to quickly orient a reader.
+
+For example, you can mention and link to the Python modules that the package provides:
+
+.. code-block:: rst
+
+   The ``verify`` package provides the :doc:`/modules/lsst.verify/index` module for gathering and submitting measurements of performance metrics to the SQuaSH database and web application.
+
+You can also mention related packages:
+
+.. code-block:: rst
+
+   Definitions of metrics and their specifications are provided separately in the :doc:`/packages/verify_metrics/index` package.
+
+If a package provides datasets (such as a test data repository), you can summarize what these datasets are for, and what APIs uses those datasets.
+
+.. _package-homepage-project-info:
+
+Project info section
+====================
+
+The Project info section is designed to summarize the nature of the package as an open source Git repository.
+See the :ref:`starter template’s example <package-homepage-template>` for how this section is presented.
```

Comment:
Do we want to say something like "A Jira component should already exist for the package, either for it specifically or for the higher-level product of which it is a part.  If one does not exist, contact a Jira administrator."?